src/main/java/org/isf/admtype/rest/AdmissionTypeController.java

```diff
@@ -73,12 +72,12 @@ public AdmissionTypeController(AdmissionTypeBrowserManager admtManager, Admissio
 	 * @throws OHServiceException
 	 */
 	@PostMapping(value = "/admissiontypes", produces = MediaType.APPLICATION_JSON_VALUE)
-	ResponseEntity<AdmissionTypeDTO> newAdmissionType(@RequestBody AdmissionTypeDTO admissionTypeDTO) throws OHServiceException {
+	ResponseEntity<?> newAdmissionType(@RequestBody AdmissionTypeDTO admissionTypeDTO) throws OHServiceException {
 		String code = admissionTypeDTO.getCode();
 		LOGGER.info("Create Admission Type {}", code);
 		AdmissionType newAdmissionType = admtManager.newAdmissionType(mapper.map2Model(admissionTypeDTO));
-		if (!admtManager.isCodePresent(code)) {
-			throw new OHAPIException(new OHExceptionMessage("Admission Type is not created."), HttpStatus.INTERNAL_SERVER_ERROR);
+		if (admtManager.isCodePresent(code)) {
+			return ResponseEntity.badRequest().body(new OHExceptionMessage("The code of Admission Type is already used."));
```

Comment:
There appears to be two different returns for something already present/being used:
<pre>
return ResponseEntity.badRequest().body(new OHExceptionMessage("XXXXX is already used."));
return ResponseEntity.internalServerError().body(new OHExceptionMessage("XXXXX is already present."));
</pre>
Shouldn't there only be one?